client/signup/steps/import-url/index.jsx

```diff
@@ -15,34 +17,48 @@ import Button from 'components/button';
 import ButtonGroup from 'components/button-group';
 import StepWrapper from 'signup/step-wrapper';
 import SignupActions from 'lib/signup/actions';
-import FormTextInputWithAction from 'components/forms/form-text-input-with-action';
-//import FormInputValidation from 'components/forms/form-input-validation';
-import { fetchIsSiteImportable, setNuxUrlInputValue } from 'state/importer-nux/actions';
+import FormTextInput from 'components/forms/form-text-input';
+import FormInputValidation from 'components/forms/form-input-validation';
+import { setNuxUrlInputValue, setValidationMessage } from 'state/importer-nux/actions';
 import {
 	getNuxUrlInputValue,
 	getSiteDetails,
+	getUrlInputValidationMessage,
 	isUrlInputDisabled,
 } from 'state/importer-nux/temp-selectors';
 
 const debug = debugFactory( 'calypso:signup-step-import-url' );
 
-const DEBOUNCE_CHECK_INTERVAL = 1200;
+const VALIDATION_INTERVAL = 1200;
 
 const normalizeUrlForImportSource = url => {
 	// @TODO sanitize? Prepend https:// ..?
 	return url;
 };
 
+const isValidUrl = ( value = '' ) => {
+	const { protocol } = parseURL( value );
+	const withProtocol = protocol ? value : 'http://' + value;
+	const { hostname } = parseURL( withProtocol );
+
+	return isWebUri( withProtocol ) && indexOf( hostname, '.' ) > 0;
+};
```

Comment:
Because we're calling this function on the change event of the input, it's going to highlight an error before I've managed to finish typing a URL. That's why I'm getting an error when I just enter `h`.

Reply:
Hi @andfinally, I just want to check here... Is this happening immediately after typing 'h' or with after an interval (as in, after the debounce)?

Reply:
Hi Spen, it might be after the debounce, because it happens about one second after I stop typing "h". It means I'm liable to get the warning if I pause while typing my URL.